Dockerfile

```diff
@@ -0,0 +1,89 @@
+# syntax=docker/dockerfile:1
+
+#	This Source Code Form is subject to the terms of the Mozilla Public License, 
+#	v. 2.0. If a copy of the MPL was not distributed with this file, You can 
+#	obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under 
+#	the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+#	
+#	Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS 
+#	graphic logo is a trademark of OpenMRS Inc.
+
+### Build Stage
+FROM maven:3.8-jdk-11 as build
+WORKDIR /app
+
+# Copy source
+COPY . .
+
+# Build the app using maven cache
+RUN --mount=type=cache,target=/root/.m2 mvn clean package -DskipTests
+
+### Run stage
+FROM tomcat:7-jdk8-adoptopenjdk-hotspot
```

Comment:
Just a thought,  is it possible to make tomcat and JDK configurable, or maybe we should avail different image flavors i.e `openmrs-core:tc9-jdk11`, `openmrs-core:tc10-jdk14`

Reply:
When we decide to provide such flavours of images, we need to make sure it is compatible and test accordingly, which is not free... I don't think we can afford that and we should focus on one tomcat version. It should not be 7 since it reached the end of line. Ideally we pick 10, if openmrs-core runs on it correctly. I will also check Tomcat version in spring-boot since we may want to benefit from their dev tools at some point.

Reply:
I found this thread: https://talk.openmrs.org/t/openmrs-update-to-latest-java-tomcat-and-mysql-versions/36505/3 which implies 10.x is not supported yet. I will upgrade to 9.x.

Reply:
yeah 9.x is more suitable and stable for the current openmrs versions ,,,thanks raf

Reply:
9.x would be the right maximum version currently.